.github/workflows/build-addon.yml

```diff
@@ -19,9 +19,15 @@ jobs:
       - uses: actions/checkout@v2
         with: 
           submodules: recursive 
-      - run: rm -rf .git/
       - run: git config --global user.email "tristanteufel@googlemail.com"
```

Comment:
maybe we dont even need git config anymore if we dont delete the .git/ folder?

Reply:
I am not sure. You do this to push to the release repo under your name right? I assume the `.git` folder of this repo does not have an user.email etc. preconfigured (I assume the github workflow checkout is like a public git clone) and the runner does not have a global git config either. So it would not know your name right? But, the token to push to the release repo could contain your author info already right (maybe depends on the kind of token)?  I would assume one would need the config here, but I am not sure.